README.md

```diff
@@ -73,6 +73,20 @@ custom:
     skipInstallNodeTracer: true # defaults to false
 ```
 
+In case you are using ES Modules for Lambda handlers.
+
+```yml
+provider:
+  name: aws
+  runtime: nodejs14.x
+
+custom:
+  lumigo:
+    token: <YOUR TOKEN GOES HERE>
+    nodeUseModule: true
```

Comment:
Yes, I was a bit unsure. I ended up with only `nodeUseModule`, since the way to notify node is by using `type="module"` in package.json -- but do I think this is bit too simplistic. But if we add to it, maybe `nodeUseESModule`  without the versioning nr would be more aligned with how these modules are referenced?